app/vmalert/rule/alerting.go

```diff
@@ -446,7 +446,7 @@ func (ar *AlertingRule) exec(ctx context.Context, ts time.Time, limit int) ([]pr
 			a.KeepFiringSince = time.Time{}
 			continue
 		}
-		a, err := ar.newAlert(m, ls, start, qFn)
+		a, err := ar.newAlert(m, ls, ts, qFn)
```

Comment:
hha, we actually correct it below
https://github.com/VictoriaMetrics/VictoriaMetrics/blob/15a15e5b99eea8962448028f00498d6619663c2c/app/vmalert/rule/alerting.go#L456
now we can remove it, so it's a fix but isn't a bug)